Include/ast.h

```diff
@@ -15,6 +15,10 @@ PyAPI_FUNC(mod_ty) PyAST_FromNodeObject(
     PyCompilerFlags *flags,
     PyObject *filename,
     PyArena *arena);
+PyAPI_FUNC(PyObject *) PyAST_UnicodeFromAstExpr(
```

Comment:
Actually, since this functions supports only AST corresponding to expressions, `PyAST_AsUnicode` is not a good name. Maybe just add an underscore to `PyAST_UnicodeFromAstExpr`? Or `_PyAST_ExprAsUnicode`?

Reply:
OK, I like _PyAST_ExprAsUnicode.